### PR TITLE
[MetaSchedule] Handle deserializing empty string RVs in trace

### PR DESCRIPTION
**Pull request by AndrewZhaoLuo**
It seems sometimes we can have random variables with names such as "" (this is not the empty string, but two quotation marks). 

I believe this is suppose to hit "Case 2" in `TranslateInputRVs` and become an empty string but this does not seem to be the case.

Unsure if having an RV as "" may be problematic, if it is, this will certainly hide another bug

cc @junrushao1994

### CODE REVIEW

**Review by zxybazh — COMMENTED** [review]
The fix looks reasonable, just want to make sure the tests make sense. Thanks @AndrewZhaoLuo for sending this patch.

**Comment by AndrewZhaoLuo** [comment]
Oops, I made a branch in main repo on accident. Please go to https://github.com/apache/tvm/pull/12481

File: tests/python/unittest/test_tir_schedule_trace.py [hunk 1]
```diff
@@ -275,5 +284,39 @@ def test_apply_json_to_schedule_1():
...
+        insts=[
+            _make_get_block(name="B", output=b0),
+            _make_annotate(block=b0, annotation=annotation),
+        ],
+        decisions={},
+    )
+    json_obj = trace.as_json()
+    sch = tir.Schedule(elementwise, debug_mask="all")
+    Trace.apply_json_to_schedule(json_obj, sch)
+    tvm.ir.assert_structural_equal(elementwise_inlined, sch.mod["main"])
```
[the remark this answers is not in the document]
Comment (AndrewZhaoLuo) [reply tests/python/unittest/test_tir_schedule_trace.py:304]: I just want to test annotation which had the bug. I've changed this to compare against the proper mod.

File: tests/python/unittest/test_tir_schedule_trace.py [hunk 2]
```diff
@@ -275,5 +284,39 @@ def test_apply_json_to_schedule_1():
...
+
+def test_apply_annotation_from_json():
+    # Something reasonable
+    _test_apply_annotation_trace_from_json("SSRSSR")
+
+    # The empty string
+    _test_apply_annotation_trace_from_json("")
+
+    # A string of two quotation marks
+    _test_apply_annotation_trace_from_json('""')
```
[the remark this answers is not in the document]
Comment (AndrewZhaoLuo) [reply tests/python/unittest/test_tir_schedule_trace.py:315]: This is for completeness

**Pull request closed without merging**